### PR TITLE
Prevent throwing RuntimeException when invalid proto is received

### LINKED ISSUE #163: Ingestion retrying an invalid FeatureRow endlessly
## Expected Behavior 
Ingestion should discard invalid FeatureRow and log it into error log

## Current Behavior
Invalid timestamp causing unhandled exception which will be retried by ingestion endlessly.

## Steps to reproduce
Run ingestion and produce FeatureRow having invalid event timestamp (`seconds` is outside of range [-62,135,596,800, +253,402,300,799] or `nanos`  is outside of range [0, +999,999,999].

### Specifications

- Version: master branch
- Subsystem: Ingestion

## Possible Solution
Add check during Validate FeatureRow

### PR DESCRIPTION
**Pull request by pradithya**
Follow up to fix #163

### CODE REVIEW

File: ingestion/src/main/java/feast/store/errors/json/JsonFileErrorsWrite.java [hunk 1]
```diff
@@ -39,18 +39,24 @@
...
+                    (rowExtended) -> {
+                      try {
+                        return KV.of(
+                            rowExtended.getRow().getEntityName(),
+                            JsonFormat.printer()
+                                .omittingInsignificantWhitespace()
+                                .print(rowExtended));
+                      } catch (InvalidProtocolBufferException e) {
+                        return KV.of(
+                            rowExtended.getRow().getEntityName(),
```
Comment (thirteen37) [inline ingestion/src/main/java/feast/store/errors/json/JsonFileErrorsWrite.java]: Are we assuming that `rowExtended.getRow().getEntityName()` was not the cause of the exception?

Comment (pradithya) [reply ingestion/src/main/java/feast/store/errors/json/JsonFileErrorsWrite.java]: Yes, at worst, that line will return empty string.

**Pull request merged**